cluster-stamp.json

```diff
@@ -953,20 +967,546 @@
+                            "name": "Metric1",
+                            "operator": "GreaterThan",
+                            "threshold": 80.0,
+                            "timeAggregation": "Average"
+                        }
+                    ],
+                    "odata.type": "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"
+                },
+                "description": "This alert monitors persistent volume utilization.",
+                "enabled": true,
+                "evaluationFrequency": "PT1M",
+                "scopes": [
+                    "[resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))]"
+                ],
+                "severity": 3,
+                "targetResourceType": "microsoft.containerservice/managedclusters",
+                "windowSize": "PT5M"
+            }
+        },
+        {
+            "type": "Microsoft.Insights/metricAlerts",
+            "location": "global",
+            "apiVersion": "2018-03-01",
+            "name": "[concat('Pods not in ready state for ', variables('clusterName'), ' CI-8')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))]",
+                "[resourceId('Microsoft.OperationsManagement/solutions',variables('containerInsightsSolutionName'))]"
+            ],
+            "properties": {
+                "actions": [],
+                "criteria": {
+                    "allOf": [
+                        {
+                            "criterionType": "StaticThresholdCriterion",
+                            "dimensions": [
+                                {
+                                    "name": "controllerName",
+                                    "operator": "Include",
+                                    "values": [
+                                        "*"
+                                    ]
+                                },
+                                {
+                                    "name": "kubernetes namespace",
+                                    "operator": "Include",
+                                    "values": [
+                                        "*"
+                                    ]
+                                }
+                            ],
+                            "metricName": "PodReadyPercentage",
+                            "metricNamespace": "Insights.Container/pods",
+                            "name": "Metric1",
+                            "operator": "LessThan",
+                            "threshold": 80.0,
+                            "timeAggregation": "Average"
+                        }
+                    ],
+                    "odata.type": "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"
+                },
+                "description": "This alert monitors pods in the ready state.",
```

Comment:
although this is true, we are looking forward to alert when a percentage of pods are NOT ready if I am not wrong.

If I'm about correct, it would be great if we can improve a bit the description.

Reply:
Sorry, I didn't come up with that description, just took it as it was already created.  I'll change the wording on here though to reflect your thoughts.

Reply:
Updated.  Thanks, this is indeed much clearer.  Kinda silly how it was before :P

Reply:
<3